src/lava/magma/core/model/py/connection.py

```diff
@@ -726,7 +774,7 @@ def _compute_trace_histories(self) -> typing.Tuple[np.ndarray, np.ndarray]:
 
         # most naive algorithm to decay traces
```

Comment:
Is this a hidden todo? If so, create an issue for developing a more sophisticated algorithm. If not, just remove the comment; also in the other place where it showed up, below.